beard/clustering/wrappers.py

```diff
@@ -36,8 +36,8 @@ class ScipyHierarchicalClustering(BaseEstimator, ClusterMixin):
 
     def __init__(self, method="single", affinity="euclidean",
                  threshold=None, n_clusters=None, criterion="distance",
-                 depth=2, R=None, monocrit=None, scoring=None,
-                 scoring_data=None):
+                 depth=2, R=None, monocrit=None, unsupervised_scoring=None,
+                 supervised_scoring=None, scoring_data=None):
```

Comment:
The only thing that puzzles me in this PR is the number of parameters passed to this constructor. A function that takes 11 arguments may be complicated to use.